grpc.go

```diff
@@ -336,6 +336,11 @@ func (a *agentGRPC) CreateContainer(ctx context.Context, req *pb.CreateContainer
 			},
 			{
 				Type: configs.NEWPID,
+				// In case the path is empty because we
+				// don't expect the containers to share
+				// the same PID namespace, a new PID ns
+				// is going to be created.
+				Path: a.sandbox.sharedPidNs.path,
```

Comment:
That's the point, in that case, we expect the container to be run inside the given PID ns. Otherwise setting NEWPID type with an empty Path will start the container in a new PID ns.